package.json

```diff
@@ -93,7 +93,7 @@
       "git add"
     ],
     "*.scss": [
-      "stylelint --syntax=scss",
```

Comment:
@lkraav Is there a possibility to make the rules less restrictive or make it automatically fix complaints on commit? 
With this set of rules `stylelint` complains about things like:
`background is defined before color in css`, `nested id elements in css` etc. which are not automatically fixed and require to manually change all the redundant complaints, which is a big waste of time. My IDE (PhpStorm) doesn't even highlight those problems (probably too minor to look at).